nvflare/app_common/app_constant.py

```diff
@@ -102,6 +102,7 @@ class AppConstants(object):
     CROSS_VAL_SERVER_MODEL = "_cross_val_server_model_"
     CROSS_VAL_CLIENT_MODEL = "_cross_val_client_model_"
     PARTICIPATING_CLIENTS = "_particpating_clients_"
+    CONTRIBUTIONS = "_client_contributions_"
```

Comment:
What is this used for? If not used, let's remove.

Reply:
It's used by an example. I moved it inside the example.